ME2Randomizer/Classes/Randomizers/ME2/Misc/RTexts.cs

```diff
@@ -150,7 +155,7 @@ private static void UwuifyTalkFile(TalkFile tf, bool keepCasing, List<int> skipI
                             sb.Append(keepCasing ? strData[i] : char.ToLower(strData[i]));
                         }
                     }
-                    else if (currentChar == '!')
+                    else if (currentChar == '!' & !addReactions)
```

Comment:
This should be &&. & is a bitwise operator, like 0100 & 0111 will yield 0100 (only the third bit is set in both). You're looking for the logic &&, which is 'and'.

Reply:
How the heck was it working with the wrong operator everywhere? Fixed all of these.